### PR TITLE
kubelet: switch deprecated --config flag to --kubeconfig

### PR DESCRIPTION
**Pull request by MaximilianMeister**
this needs to be done during upgrade to 2.0 otherwise kubelet
wont start

Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

File: admin-node-setup.sh [hunk 1]
```diff
@@ -48,3 +48,6 @@ cp $tmp_dir/public.yaml $kube_dir
 cp $tmp_dir/private.yaml $kube_dir
 
 rm -rf $tmp_dir
+
+# switch deprecated --config flag in kubelet
+sed -i -e "s/--config/--kubeconfig/g" /etc/kubernetes/kubelet
```
Comment (kiall) [inline admin-node-setup.sh]: should be moved to `--pod-manifest-path`

Comment (MaximilianMeister) [reply admin-node-setup.sh]: updated

File: admin-node-setup.sh [hunk 2]
```diff
@@ -48,3 +48,6 @@ cp $tmp_dir/public.yaml $kube_dir
 cp $tmp_dir/private.yaml $kube_dir
 
 rm -rf $tmp_dir
+
+# switch deprecated --config flag in kubelet
+sed -i -e "s/--config/--pod-manifest-path/g" /etc/kubernetes/kubelet
```
Comment (ereslibre) [inline admin-node-setup.sh]: Nit: I'd include `--config=` and `--pod-manifest-path=`. Just to make sure we don't replace anything starting with `--config`. It probably doesn't exist (don't have a cluster at hand to check), but just for sanity.

**Comment by kiall** [comment]
https://bugzilla.suse.com/show_bug.cgi?id=1062011

File: admin-node-setup.sh (hunk 2 above)
Comment (MaximilianMeister) [reply admin-node-setup.sh]: done

**Comment by MaximilianMeister** [comment]
fixed @ereslibre comment and added the bug reference

**Comment by MaximilianMeister** [comment]
included in https://github.com/kubic-project/caasp-container-manifests/pull/123

**Pull request closed without merging**